xlml/apis/task.py

```diff
@@ -321,15 +321,17 @@ class XpkTask(BaseTask):
   task_gcp_config: gcp_config.GCPConfig
   task_metric_config: Optional[metric_config.MetricConfig] = None
 
-  def run(self) -> DAGNode:
+  def run(
+      self, shared_gcs_location: Optional[airflow.XComArg] = None
```

Comment:
What do you think about making this kwarg-only?

```python
def run(self, *, gcs_path: Optional[airflow.XComArg] = None):
```

Then to use in `run` it must be specified by name, e.g.

```python
config.get_flax_resnet_xpk_config(...).run(gcs_path=shared_gcs_location)
```

Reply:
I don't have a strong preference here, since it only has 1 param for both run() and run_model(). May I know the reason you suggest this?

Reply:
I like the `kwarg`-only idea for this since it's not an intuitive parameter for `run` IMO

Reply:
Cool! Both of you like the idea, I will just update it. Thanks!